app/controllers/albums.js

```diff
@@ -0,0 +1,15 @@
+const logger = require('../logger');
+const axios = require('axios');
+
+module.exports = {
+  list(req, res, next) {
+    axios.get('https://jsonplaceholder.typicode.com/albums').then(function(response) {
```

Comment:
Let's move the url to a config.

Also it would be nice to remove the external api logic to another file and not have it all in the controller.

If you want you could take a look a the following [document](https://docs.google.com/document/d/1ufn2NBFJkGyDmfDQCSWjBHrAQku0ZfZemc8aDV97qk8/edit?usp=sharing) where we talk about this type of responsabilities distribution.

Reply:
Changed

Reply:
The Url was moved to config in #8